src/simpla-text-editor.html

```diff
@@ -107,9 +205,10 @@
         this.is = 'simpla-text-toolbar';
 
         this.properties = {
+
           tools: {
             type: Object,
-            value: INITIAL_TOOLS
+            value: () => INITIAL_TOOLS
```

Comment:
Not a big deal given this is a singleton, but best practice:

So this should be reverted. The way in this PR, the function will always return `INITIAL_TOOLS`, which means it'll always be the same instance, so `INITIAL_TOOLS` will be shared by all the instances. If `INITIAL_TOOLS` is a function which returns a new instance then it'll be fine - but it's a bad name for it. I should have named it something like `getInitialTools`.

But yeah - I'd revert to a function called `getInitialTools`, or just have the function return a fresh object each call.